geode-gfsh/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DiskStoreCommandsDUnitTest.java

```diff
@@ -519,6 +520,25 @@ public void offlineDiskStoreCommandShouldNotCreateFolderIfDiskStoreDoesNotExist(
     assertThat(Files.exists(nonExistingDiskStorePath)).isFalse();
   }
 
+  @Test
+  @Parameters({"compact offline-disk-store", "describe offline-disk-store",
+      "upgrade offline-disk-store", "validate offline-disk-store",
+      "alter disk-store --region=testRegion --enable-statistics=true"})
+  public void offlineDiskStoreCommandShouldNotPassIfDiskStoreFileDoesNotExist(
```

Comment:
Can you change the test name to something more meaningful?, maybe `offlineDiskStoreCommandShouldFailWhenDiskStoreFileDoesNotExist`?.